tests/unit/proto/test_protobuf.py

```diff
@@ -1,31 +1,29 @@
-import unittest
+import pytest
 
 from jina.proto.jina_pb2 import Document
-from tests import JinaTestCase
 from google.protobuf.json_format import MessageToJson, Parse
 
+@pytest.fixture(scope='function')
```

Comment:
Is it necessary to set up the scope here? The default value is `function`. 
https://docs.pytest.org/en/stable/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session

Reply:
You're right. It's my personal habit to be explicit https://www.python.org/dev/peps/pep-0020/